board-together/FE-Board-Together
# Combined/feature/component architecture

#### What's this PR do?
This merge will set up the React component file architecture, and add routes to the main `App`.

#### Where should the reviewer start?
- In CLI run `git fetch` and then `git checkout combined/feature/component-architecture`
- Look at code and file structure in IDE of preference
- In CLI run `npm start` which will automatically open the application in the browser.
#### How should this be manually tested?
In the URL bar, add `/login` or `/search-results/{search term of choice}` to see how the application is being routed.
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/74210902/216846312-a1b297e0-1004-4c2b-b285-e23df5c6157e.png)
![image](https://user-images.githubusercontent.com/74210902/216846359-76d2ea39-0f2f-4b37-a7e5-e3679cad9f99.png)
